### PR TITLE
New Sign In Screen with Sidebar

### PR DESCRIPTION
**Pull request by youngkidwarrior**
Newest SignIn designs in line with brand redesign. Retaining gold color pallete for now until theme is updated to green colors

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #80** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/80?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #81** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/80?utm_source=stack-comment).

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Jan 24, 10:08 PM**: [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/80) rebased this pull request as part of a merge.

**Review by 0xBigBoss — COMMENTED** [review]
Maybe we should make it responsive first? 

Screenshot 2024-01-25 at 22 19 15

**Comment by youngkidwarrior** [comment]
See #94

**Comment by youngkidwarrior** [comment]
@0xBigBoss Can I get an approval on this and #81 so i can start merging this stack

File: packages/app/features/auth/sign-in-screen.tsx
[the remark this answers is not in the document]
Comment (youngkidwarrior) [reply packages/app/features/auth/sign-in-screen.tsx:1]: The narrow screen styles are handled in a future PR they is stacked on this one #94

[the remark this answers is not in the document]
Comment (youngkidwarrior) [reply packages/app/features/auth/sign-in-screen.tsx:1]: @0xBigBoss can I get an approve here to start merging this stack?

**Review by 0xBigBoss — DISMISSED** [review]

**Comment by youngkidwarrior** [comment]
@0xBigBoss shit, looks like rebasing this on top of main caused your approval to stale :/

**Comment by 0xBigBoss** [comment]
> @0xBigBoss shit, looks like rebasing this on top of main caused your approval to stale :/
> 
> 

Igotchu. Approved.

**Pull request merged**